### PR TITLE
Improved warning message when weaver cannot be loaded

### PR DESCRIPTION
**Pull request by GeertvanHorrik**
Based on the issues noted here:

https://github.com/Fody/Costura/issues/115

### CODE REVIEW

File: FodyIsolated/DelegateBuilders/DelegateBuilder.cs [hunk 1]
```diff
@@ -19,39 +20,68 @@ public static WeaverDelegate GetDelegateHolderFromCache(this Type weaverType)
...
+            SetReferences = weaverType.BuildPropertySetDelegate<string>("References"),
+            SetReferenceCopyLocalPaths = weaverType.BuildPropertySetDelegate<List<string>>("ReferenceCopyLocalPaths"),
+            SetSolutionDirectoryPath = weaverType.BuildPropertySetDelegate<string>("SolutionDirectoryPath"),
+            SetProjectDirectoryPath = weaverType.BuildPropertySetDelegate<string>("ProjectDirectoryPath"),
+            SetDefineConstants = weaverType.BuildPropertySetDelegate<List<string>>("DefineConstants"),
+            ConstructInstance = weaverType.BuildConstructorDelegate()
+        };
+    }
+
+    private static string GetAssemblyVersion(Assembly assembly, int separatorCount = 3)
```
Comment (SimonCropp) [inline FodyIsolated/DelegateBuilders/DelegateBuilder.cs]: why this method instead of `typeof(ModuleDefinition).Assembly.GetName().Version`?

Comment (GeertvanHorrik) [inline FodyIsolated/DelegateBuilders/DelegateBuilder.cs]: Because I am a noob ;-)

Comment (SimonCropp) [inline FodyIsolated/DelegateBuilders/DelegateBuilder.cs]: lol. ok. rest looks good. want me to merge manually? or do u want to fix the pr?

Comment (GeertvanHorrik) [inline FodyIsolated/DelegateBuilders/DelegateBuilder.cs]: Fixing pr.

**Comment by GeertvanHorrik** [comment]
Done.

**Comment by SimonCropp** [comment]
assuming u dont need this deployed right away? since u have already solved the main issue?

**Comment by GeertvanHorrik** [comment]
Yep, this is just for future usage (so it reminds us why something might be broken).

**Pull request merged**